darts/models/forecasting/nhits.py

```diff
@@ -83,7 +85,10 @@ def __init__(
         self.batch_norm = batch_norm
         self.dropout = dropout
 
-        self.activation = nn.ReLU()  # TODO: make configurable?
+        if activation == "relu":
+            self.activation = nn.ReLU()
+        else:
```

Comment:
I would check that the name provided is among the known activations and raise an error otherwise.